source/styleguide/components.liquid

```diff
@@ -0,0 +1,127 @@
+---
+title: Components
+---
+
+<h2>Closed</h2>
+
+<dl class="sl-c-description-list sl-c-description-list--horizontal">
+  <div>
+    <dt>Dart Sass</dt>
+    <dd>✓</dd>
+  </div>
+  <div>
+    <dt>Ruby Sass</dt>
+    <dd>since 3.5.0</dd>
+  </div>
+  <div><a href="#">▶︎</a></div>
+</dl>
+
+<h2>Open</h2>
+
+<dl class="sl-c-description-list sl-c-description-list--horizontal">
+  <div>
+    <dt>Dart Sass</dt>
+    <dd>✓</dd>
+  </div>
+  <div>
+    <dt>Ruby Sass</dt>
+    <dd>since 3.5.0</dd>
+  </div>
+  <div><a href="#">▼</a></div>
+</dl>
+
+<div class="sl-c-callout">
+  <p>{% lorem 'paragraph' %}</p>
+</div>
+
+<h2>Alerts</h2>
+
+<div class="sl-c-alert">
+  <div class="sl-l-container">
+    <p>
+      <strong>Alert</strong>
+      {% lorem 'sentence' %}
+    </p>
+  </div>
+</div>
+
+<div class="sl-c-alert sl-c-alert--info">
```

Comment:
@stacyk Something is breaking with the background color of this `.sl-c-alert--info`. Maybe that's fixed in another branch?

Reply:
@jgerigmeyer It was worked on in another PR that is now in this branch.